### ISSUE TITLE
Automated (bot) accounts are being mistakenly treated as verified

### ISSUE THREAD

**Issue by Anyasia** [kheina-com/Blue-Blocker#157]
Apologies for submitting yet another issue, but I've also noticed that Twitter accounts flagged as bots are being queued for blocking:—

```
[Blue Blocker] queued Otter(s) an Hour (@OtterAnHour) for a block due to Twitter Blue verified.
[Blue Blocker] queued Raccoons Hourly (@raccoonhourly) for a block due to Twitter Blue verified.
[Blue Blocker] queued HourlyYeen (@HourlyYeen) for a block due to Twitter Blue verified.
[Blue Blocker] queued Garfield and Friends Screens (@GarfieldScreens) for a block due to Twitter Blue verified.
[Blue Blocker] queued Ursula K. Le Guin bot (@UrsulaBot) for a block due to Twitter Blue verified.
```

Not sure whether this was happening prior to v0.3.1 or whether it's new to that version. No relevant logs in Firefox' service worker console.

Thanks again. ♥

**Comment by DanielleMiu** [comment]
it looks like this is caused by the bot label itself and a previous fix that went out to not miss business verified users.

https://github.com/kheina-com/Blue-Blocker/blob/main/src/shared.ts#L374

for now, you can skip these by using the "skip users verified via businesses" or by adjusting your follower threshold

**Comment by DanielleMiu** [comment]
should be fixed in v0.3.2
